src/api/commands/osu/farm/farm.ts

```diff
@@ -147,7 +147,7 @@ export async function farmgraph(message: any, args: any, prefix: any, mode: any)
             break;
     }
 
-    let fields = "Number of top 100 scores set at specific hour";
+    let fields = "Number of top 100 scores for this specific case:";
```

Comment:
this is a very wrong way of doing this.

sicne we have the information what graph were generating we can also display a fitting field